ql/math/rounding.cpp

```diff
@@ -26,12 +26,21 @@
 
 namespace QuantLib {
 
+    static inline Real fast_pow10(Integer n) {
+        // supporting precision up to double 1e-16 (15.9 digits)
+        constexpr Real pow10_map[] = {1.0E0,  1.0E1,  1.0E2,  1.0E3,  1.0E4,  1.0E5,
```

Comment:
Add `const static` to this. Otherwise, the array is copied from static memory into the stack before access. (clang manages to optimize this away, but apparently gcc does not: https://godbolt.org/z/WqWWzbqWY)